operator/config/rbac/kustomization.yaml

```diff
@@ -15,6 +15,11 @@ resources:
 - forklift-controller_role.yaml
 - forklift-controller_role_binding.yaml
 
+# volume-data-source-validator roles
+- volume-data-source-validator/service_account.yaml
+- volume-data-source-validator/role.yaml
+- volume-data-source-validator/role_binding.yaml
```

Comment:
these for some reason don't apply.. @mnecas thoughts?

Reply:
we can do without them. the user may deploy it. we only need the `VolumePopulator` CRD.